platform/core.output2/src/org/netbeans/core/output2/NbIOProvider.java

```diff
@@ -214,7 +214,17 @@ public Lookup getIOLookup(InputOutput io) {
     @Override
     public void resetIO(InputOutput io) {
         if (io instanceof NbIO) {
-            ((NbIO) io).reset();
+            if(io.getOut() instanceof NbWriter) {
+                try {
+                    // NbWriter does OutWriter fixup,
+                    // then NbWriter invokes ((NbIO) io).reset()
+                    io.getOut().reset();
+                } catch(IOException ex) {
+                    Exceptions.printStackTrace(ex);
```

Comment:
This seems to be the way errors are handled in this class, I guess.

Reply:
This will probably show a dialog to the user. If you just want to print errors to console, consider `Exceptions.attachSeverity` first.